java/client/src/main/java/glide/api/models/commands/SortBaseOptions.java

```diff
@@ -0,0 +1,107 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.GenericClusterCommands;
+import glide.api.commands.GenericCommands;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * Optional arguments to {@link GenericClusterCommands#sort(String, SortClusterOptions)}, {@link
+ * GenericClusterCommands#sortReadOnly(String, SortClusterOptions)}, {@link
+ * GenericClusterCommands#sortStore(String, String, SortClusterOptions)}, {@link
+ * GenericCommands#sort(String, SortOptions)}, {@link GenericCommands#sortReadOnly(String,
+ * SortOptions)}, and {@link GenericCommands#sortStore(String, String, SortOptions)}
+ *
+ * @see <a href="https://redis.io/commands/sort/">redis.io</a> and <a
+ *     href="https://redis.io/docs/latest/commands/sort_ro/">redis.io</a>
+ */
+@SuperBuilder
+public abstract class SortBaseOptions {
+    /**
+     * <code>LIMIT</code> subcommand string to include in the <code>SORT</code> and <code>SORT_RO
+     * </code> commands.
+     */
+    public static final String LIMIT_COMMAND_STRING = "LIMIT";
+
+    /**
+     * <code>ALPHA</code> subcommand string to include in the <code>SORT</code> and <code>SORT_RO
+     * </code> commands.
+     */
+    public static final String ALPHA_COMMAND_STRING = "ALPHA";
+
+    /** <code>STORE</code> subcommand string to include in the <code>SORT</code> command. */
+    public static final String STORE_COMMAND_STRING = "STORE";
+
+    /**
+     * Limiting the range of the query by setting offset and result count. See `Limit` class for more
+     * information.
+     */
+    private final Limit limit;
+
+    /** Options for sorting order of elements. */
+    private final OrderBy orderBy;
+
+    /**
+     * When <code>true</code>, sorts elements lexicographically. When <code>false</code> (default),
+     * sorts elements numerically. Use this when the list, set, or sorted set contains string values
+     * that cannot be converted into double precision floating point numbers.
+     */
+    private final boolean alpha;
```

Comment:
could you call alpha in the builder like StreamReadGroupOptions.noack()?  
I can show you how to include the builder code if you like... it's... annoying in a SuperBuilder